perfkitbenchmarker/network.py

```diff
@@ -51,12 +51,14 @@ def GetFirewall(cls):
         benchmark_spec.firewalls[key] = cls()
       return benchmark_spec.firewalls[key]
 
-  def AllowPort(self, vm, port):
+  def AllowPort(self, vm, start_port, end_port=None):
```

Comment:
I actually did rebase off of master after that PR merged, but I realized that I forgot to change the `BaseVirtualMachine` and `BaseFirewall` APIs. I figured it'd be alright to slip that into this PR, but if you want I can make a separate PR.